hyperspy/drawing/_markers/point.py

```diff
@@ -87,7 +87,7 @@ def update(self):
                                  self.get_data_position('y1')])
         self.marker._sizes = [self.get_data_position('size')]
 
-    def plot(self):
+    def plot(self, fig_draw_animated=True):
```

Comment:
I added an `update_plot` argument to `marker.plot()` and `marker.close()`, if `update_plot=False` it does not call `hspy_fig.draw_animated()`.